Current/Invalid Objects.sql

```diff
@@ -0,0 +1,59 @@
+USE [SQLCopTests]
+GO
+SET ANSI_NULLS ON
+GO
+SET QUOTED_IDENTIFIER ON
+GO
+--  Test to identify Invalid Stored Procedures and Views
+--  If any invalid objects are found then fail the test with list of affected objects
+--	If you require a comprehensive list of Invalid Objects you can use the SQL Prompt find invalid objects functionality: https://documentation.red-gate.com/sp/sql-refactoring/refactoring-an-object-or-batch/finding-invalid-objects
+
+--	Written by Chris Unwin 17/09/2019
+
+ALTER PROCEDURE [SQLCop].[test Invalid Objects]
+AS
+BEGIN
+
+    SET NOCOUNT ON;
```

Comment:
Everything up to and including this `SET NOCOUNT ON;` line should be replaced with the following:

```sql
IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'[SQLCop].[test Invalid Objects]') AND type in (N'P', N'PC'))
DROP PROCEDURE [SQLCop].[test Invalid Objects]
GO

CREATE PROCEDURE [SQLCop].[test Invalid Objects]
AS
BEGIN
    --  Test to identify Invalid Stored Procedures and Views
    --  If any invalid objects are found then fail the test with list of affected objects
    --  If you require a comprehensive list of Invalid Objects you can use the SQL Prompt find invalid objects functionality:
    --  https://documentation.red-gate.com/sp/sql-refactoring/refactoring-an-object-or-batch/finding-invalid-objects

    --  Written by Chris Unwin 17/09/2019
    SET NOCOUNT ON
```

Reply:
Have done so thank you!